build.sbt

```diff
@@ -1,10 +1,4 @@
 ThisBuild / organization := "com.eed3si9n"
-ThisBuild / dynverSonatypeSnapshots := true
-ThisBuild / version := {
```

Comment:
I understand why this was probably added back then, but since there isn't a ton of active development going on here, getting rid of this gets rid of having to manual update it. If you want me to add this back in, just let me know.

Reply:
Please restore this part. I don't want each snapshot to take up disk space either in cache or on my machine when I run tests.

Reply:
Sure, added back in.